lib/index.js

```diff
@@ -5,12 +5,17 @@ const path = require('path')
 const axios = require('axios')
 
 
+const api = axios.create({
```

Comment:
this requires axios as dependency, maybe have the api generic/interface and the clients will pass the implementation.